README.es.md

```
@@ -0,0 +1,477 @@
+
+## Descargar
+
+| Plataforma            | Descarga                                               |
+| --------------------- | ------------------------------------------------------ |
+| macOS (Apple Silicon) | [Descargar DMG](https://voicebox.sh/download/mac-arm)   |
+| macOS (Intel)         | [Descargar DMG](https://voicebox.sh/download/mac-intel) |
+| Windows               | [Descargar MSI](https://voicebox.sh/download/windows)   |
+| Docker                | `docker compose up`                                    |
+
+> **[Ver todos los binarios →](https://github.com/jamiepine/voicebox/releases/latest)**
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)